### ISSUE TITLE
Could not generate field 'single' with unknown type '1'.

### ISSUE THREAD

**Issue by alissonviegas** [lucascaton/enumerate_it#102]
Hi! Thanks for this nice gem.
I have the below error when generate a hash enum:

Command:
```
rails g enumerate_it:enum CivilStatus single:1 married:2 divorced:3 widower:4
```

Error:
```
Could not generate field 'single' with unknown type '1'.
```

Enviroment:
- Ubuntu 20.04
- Rails 7.0.3.1
- ruby 3.1.0

**In Rails 6 works.**
Thanks!

**Comment by lucascaton** [comment]
Thanks for reporting it, @alissonviegas!

I've released [version `3.2.3`](https://rubygems.org/gems/enumerate_it/versions/3.2.3) which includes a fix.

Here's the full release notes: https://github.com/lucascaton/enumerate_it/releases/tag/v3.2.3

**Comment by alissonviegas** [comment]
Thank you! 😉👍
